src/ucp/wireup/wireup.c

```diff
@@ -1153,38 +1150,33 @@ ucp_wireup_check_config_intersect(ucp_ep_h ep, ucp_ep_config_key_t *new_key,
         return;
     }
 
+    ucs_assert(!(ep->flags & UCP_EP_FLAG_INTERNAL));
```

Comment:
what does the flag UCP_EP_FLAG_INTERNAL mean now?

Reply:
this is used for MEMTYPE EP and REPLY EP (will be implemented)